app-web/teams-for-linux/autobuild/overrides/usr/share/applications/teams-for-linux.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Type=Application
+Name=Microsoft Teams for Linux
+Comment=Unofficial Microsoft Teams client for Linux
+GenericName=Teams
+Exec=teams-for-linux %U
+Icon=teams-for-linux
+MimeType=x-scheme-handler/msteams;
+Categories=Network;Chat;InstantMessaging;Application;
```

Comment:
`desktop-file-validate /usr/share/applications/teams-for-linux.desktop`
/usr/share/applications/teams-for-linux.desktop: warning: value "Network;Chat;InstantMessaging;Application;" for key "Categories" in group "Desktop Entry" contains a deprecated value "Application"

Consider removing `Application;` from this line.